site/content/docs/0.2/extend/icons.md

```diff
@@ -1,11 +1,736 @@
+
+[Solaris icons library]({{< param icons >}}) is a growing library of icons that are designed by [Orange Design System design team](https://system.design.orange.com/). It features an icon search tool based on categories and keywords, and allows icons to be downloaded in various formats for use in design and development. They are not open-source though and should only be used for Orange projects. Please refer to the [icons license file]({{< param icons_license >}}) for legal information.
+
+{{< callout info >}}
+You can find here the [Iconography design guidelines](https://unified-design-system.orange.com/472794e18/p/275963-iconography).
+{{< /callout >}}
+
+If you need guidance to use Solaris icons, please refer to the paragraph [use Solaris icons]({{< docsref "extend/icons#use-solaris-icons" >}}).
+
+## Icons rendering
+
+Icons are designed within a square layout to preserve consistency. Within this square, there exists a designated safety zone to guarantee that icons can be used in various sizes and contexts while maintaining alignment as intended by the designers. The dimensions of the icons encompass this safety zone, ensuring adaptability and consistency across diverse applications.
+
+<svg width="8em" height="8em" class="bg-body-secondary" aria-hidden="true">
+  <use xlink:href="/docs/{{< param docs_version >}}/assets/img/ouds-web-sprite.svg#info"/>
+</svg>
+<svg width="8em" height="8em" class="bg-body-secondary" aria-hidden="true">
+  <use xlink:href="/docs/{{< param docs_version >}}/assets/img/ouds-web-sprite.svg#download"/>
+</svg>
+<svg width="8em" height="8em" class="bg-body-secondary" aria-hidden="true">
+  <use xlink:href="/docs/{{< param docs_version >}}/assets/img/ouds-web-sprite.svg#tick"/>
+</svg>
+
+## Icons color
+
+To color icons, you can:
+- use [color utilities classes]({{< docsref "utilities/colors#colors" >}}) like `.text-brand-primary`, `.text-disabled` or `.text-status-positive` that change current color value.
+- specify a color using CSS variables like `--bs-color-content-brand-primary`, `--bs-color-content-disabled"` or `--bs-color-content-status-positive"`.
+
+For SVGs (SVG sprites, inline SVGs or CSS background SVGs), the `fill="currentColor"` attribute is required on the `<svg>` or `<symbol>` tag, if you want to change the icons color on the fly: it will make the icon inherit color from itself or its parent containers. It is set on the icons downloaded from Solaris icons finder.
+
+### Functional icons
+
+The following color utilities and CSS variables are intended for use only with functional icons, not with text, and icons should not be used on their own, but only alongside text, as functional colors do not provide sufficient contrast to ensure readability.
+
+{{< example >}}
+<div class="d-flex justify-content-center gap-md-tall">
```

Comment:
As you raised earlier, it's very hard to understand design choices about not making them accessible.